package.json

```diff
@@ -82,6 +82,7 @@
     "@patternfly/react-core": "^4.6.1",
     "@patternfly/react-icons": "^4.1.1",
     "@patternfly/react-styles": "^4.1.1",
+    "axios": "^0.19.2",
```

Comment:
Resolve Merge conflict here.

Reply:
Conflict Resolved , axios is required to fetch data from an api so it needs to be added in package.json